### ISSUE TITLE
Shape parameters not working in ask queries

### ISSUE THREAD

**Issue by acnetj** [JeroenDeDauw/Maps#492]
Version 2 Semantic maps allowed using lines and points parameters in #ask query, producing a map more than just points. 

![image](https://user-images.githubusercontent.com/10605369/48605569-fa7d8700-e931-11e8-9944-b4f1634a10a2.png)

I could nest another ask query to find the lines I am interested in and place it into the main ask query for points:

```
{{#ask:[[-Ferry::<q>[[Provider::SF Bay Ferry]] OR [[Provider::San Francisco Bay Ferry]]</q>]]|?latlong|?-Ferry|icon=Ferry20.png|format=googlemaps|width=100%|height=100%|lines=
{{#ask: [[Provider::SF Bay Ferry]][[line::+]] OR [[Provider::San Francisco Bay Ferry]][[line::+]]| ?line=|mainlabel=-|format=list|sep=;}}}}
```

**Comment by JeroenDeDauw** [comment]
Confirmed on version 5.6 with https://sandbox.semantic-mediawiki.org/wiki/Maps_issue_492

**Comment by JeroenDeDauw** [comment]
Fixed. Will be part of 6.3

**Comment by JeroenDeDauw** [comment]
Now released as 6.2.1
